### PR TITLE
Fix abi method name

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

File: src/Neo.Compiler.MSIL/Neo.Compiler.MSIL.csproj [hunk 1]
```diff
@@ -19,6 +19,7 @@
     <Product>Neo.Compiler.MSIL</Product>
     <Description>Neo.Compiler.MSIL</Description>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <LangVersion>latest</LangVersion>
```
Comment (erikzhang) [inline src/Neo.Compiler.MSIL/Neo.Compiler.MSIL.csproj]: We don't need `LangVersion` in `netcoreapp3.1` or `netstandard2.1`.

Comment (shargon) [reply src/Neo.Compiler.MSIL/Neo.Compiler.MSIL.csproj]: I was not able to compile without this line

image

Comment (erikzhang) [reply src/Neo.Compiler.MSIL/Neo.Compiler.MSIL.csproj]: Are you using the latest version of visual studio?

Comment (erikzhang) [reply src/Neo.Compiler.MSIL/Neo.Compiler.MSIL.csproj]: My version is 16.4.0

Comment (shargon) [reply src/Neo.Compiler.MSIL/Neo.Compiler.MSIL.csproj]: 16.3.10, I will update it now

Comment (shargon) [reply src/Neo.Compiler.MSIL/Neo.Compiler.MSIL.csproj]: With the last version it's not needed, if you want we can remove the line

**Comment by vncoelho** [comment]
Ok here as well 

image

image

dotnet-sdk-3.1

**Pull request merged**